doc/whats_new.rst

```diff
@@ -33,6 +33,8 @@ Changelog
 - :func:`write_raw_bids` now adds citations to the README, by `Alex Rockhill`_ (`#463 <https://github.com/mne-tools/mne-bids/pull/463>`_)
 - :func:`make_dataset_description` now has an additional parameter ``dataset_type`` to set the recommended field ``DatasetType`` (defaults to ``"raw"``), by `Stefan Appelhoff`_ (`#472 <https://github.com/mne-tools/mne-bids/pull/472>`_)
 - :func:`mne_bids.copyfiles.copyfile_brainvision` now has an ``anonymize`` parameter to control anonymization, by `Stefan Appelhoff`_ (`#475 <https://github.com/mne-tools/mne-bids/pull/475>`_)
+- :func:`mne_bids.read_raw_bids` and :func:`mne_bids.write_raw_bids` now map respiratory (``RESP``) channel types, by `Richard Höchenberger` (`#482 <https://github.com/mne-tools/mne-bids/pull/482>`_)
+- :func:`mne_bids.read_raw_bids` now tries to correctly map channel types that are not in all-uppercase letters. Since this is in violation of BIDS, a warning is raised, by `Richard Höchenberger` (`#482 <https://github.com/mne-tools/mne-bids/pull/482>`_)
```

Comment:
not sure if I would say this in the changelog ... i'd rather treat it silently. --> but let's see, we are not done discussing anyway, with Mainak's point that BIDS is agnostic at best, ambiguous at worst about casing

Reply:
Another (temporary?) solution for MNE-BIDS could also be to always change to uppercase when reading BIDS channel types, i.e.,
```python
    for ch_name, ch_type in zip(ch_names_tsv, ch_types_json):
        updated_ch_type = bids_to_mne_ch_types.get(ch_type.upper(), None)  # <-- !!!

        if updated_ch_type is not None:
            channel_type_dict[ch_name] = updated_ch_type
```

Reply:
@jasmainak WDYT?

Reply:
@sappelhoff 
I've dropped this changelog entry.

Reply:
I think we need to decide: 1) how specification treats the cases, 2) whether the validator is an accurate reflection of the specification, and 3) what we do in MNE-BIDS about it?

From a user perspective, I think it's nicer if the cases are taken care of in MNE-BIDS without complaining. E.g., through `blah.upper()` or `blah.lower()`. You have a similar situation with email addresses: you expect people to use the same casing but actually it doesn't matter. I would still have the validator be more whiny so that we are more forgiving to downstream applications that are not as careful as MNE-BIDS.